projects/1.16/assets/ender-tendril/endertendril/lang/zh_cn.json

```
@@ -1 +1,7 @@
-{}
\ No newline at end of file
+{
+  "block.endertendril.ender_tendril": "末地藤蔓",
+  "block.endertendril.ender_tendril_plant": "末地藤蔓",
+  "block.endertendril.flowering_ender_tendril": "开花的末地藤蔓",
+  "item.endertendril.ender_tendril_seed": "末地藤蔓种子",
+  "item.endertendril.tendril_pearl": "藤蔓珍珠"
```

```suggestion
  "block.endertendril.ender_tendril": "末影卷须",
  "block.endertendril.ender_tendril_plant": "末影卷须植株",
  "block.endertendril.flowering_ender_tendril": "开花的末影卷须",
  "item.endertendril.ender_tendril_seed": "末影卷须种子",
  "item.endertendril.tendril_pearl": "卷须珍珠"
```
L3是根据key来的
现在的问题是，[tendril](https://en.wikipedia.org/wiki/Tendril) 是[攀缘植物](https://en.wikipedia.org/wiki/Vine)的器官（茎），模组里是一种植物